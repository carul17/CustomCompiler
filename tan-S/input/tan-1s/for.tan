main {
	const j := 5;
	for(j from 1 to 3){
		for(i from 1 to 3) {
			if(i == 2){
				continue;
			}
			print "i is" \s i \s; 
		}
		print \n "j is" \s j \n;
	}
	
	print j \n;
}