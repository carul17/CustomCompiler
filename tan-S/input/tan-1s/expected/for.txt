i is 1 i is 3 
j is 1
i is 1 i is 3 
j is 2
i is 1 i is 3 
j is 3
5
